### PR TITLE
Fix response header supression, Rack::Utils::HeaderHash#to_hash

### PR DESCRIPTION
**Pull request by bf4**
- Fix response header suppression, Rack::Utils::HeaderHash#to_hash 
- Shell escape path

### CODE REVIEW

**Comment by bf4** [comment]
cc @r7kamura

**Comment by bf4** [comment]
cc @r7kamura

File: lib/autodoc/document.rb [hunk 1]
```diff
@@ -141,6 +141,7 @@ def response_http_version
 
     def response_header
       table = response.headers.clone
+      table = table.to_hash if table.respond_to?(:to_hash)
```
Comment (bf4) [inline lib/autodoc/document.rb]: 👀

**Comment by bf4** [comment]
☠️

**Comment by r7kamura** [comment]
@bf4 Sorry, I had overlooked this PR, and thank you for patiently explaining that to me 🙇

**Pull request merged**